cifar10_parallel/cifar10_parallel_run.py

```diff
@@ -0,0 +1,165 @@
+import time
```

Comment:
For future reference: it is better to `git mv cifar10_run.py cifar10_parallel_run.py` compared to `git rm` then `git add`